README.md

```diff
@@ -2,6 +2,18 @@
 
 Substra end-to-end (e2e) tests
 
+# A few words regarding tests design
+
+- Must be fast to run
```

Comment:
Indeed we won't be able to force all tests to be fast but I think we will need to add a "fast mode" to be able to run all the tests in less than X minutes for a quick development loop. I think that tests are already taking too much time and it will continue to increase.

I will add a bit of context if that's ok with you.

Reply:
+1 for the fast mode! And I'm all for a bit more context.